src/Components/AlbumCard/AlbumCard.js

```diff
@@ -9,7 +10,15 @@ function AlbumCard ({ title }) {
     variables: { title: title }
   })
   
-  if (loading) return 'Loading...'
+  if (loading) return (
+    <div className='card_card' data-cy='card_card'>
+      <Skeleton animation="wave" variant="rect" width={250} height={250} />
```

Comment:
These `Skeleton` components are really handy - I'll have to use these in the future.